include/glow/Graph/Graph.h

```diff
@@ -838,6 +838,18 @@ class Function final : public Named {
   SaveNode *createSave(llvm::StringRef name, NodeValue input,
                        Placeholder *output);
 
+  /// Creates a SendNode named \p name that sends \p input to \p address. A
+  /// ReceiveNode that listens on the same address will receive the sent data.
+  /// The node also writes the input to a Placeholder.
+  SendNode *createSend(llvm::StringRef name, NodeValue input,
+                       Placeholder *address);
```

Comment:
Do we really want to use addresses for this? At the time of execution on the device, placeholders may not even exist on the device side. 

May be some kind of a tag would be better? A tag could be e.g. uint64_t (or bigger) and it could be mapped by the backend/DeviceMgr or even by the runtime on device to specific buffers.

Reply:
I should have chosen a better name here. This "`address`" is really a token that the backend interprets however it wants. The value itself is used only to indicate which `Send` and `Receive` nodes are paired at runtime.